src/pgroonga.c

```diff
@@ -6119,6 +6127,14 @@ PGrnSearchBuildCondition(Relation index, ScanKey key, PGrnSearchData *data)
 		PGrnSearchBuildConditionIn(data, key, targetColumn, attribute);
 		return;
 	}
+	if (PGrnSearchIsMatchIn(key))
```

Comment:
Is it possible that we expand the existed logic of `PGrnSearchIsInCondition` instead of defining `PGrnSearchIsMatchIn`?

Reply:
I don't unify this logic.
Because `IN` and `IN + Match` is different.

Reply:
Thank you so much. I understood.